src/SelectField/SelectField.jsx

```diff
@@ -20,9 +20,15 @@ function getStyles(props) {
 }
 
 const SelectField = React.createClass({
+  displayName: 'SelectField',
 
   propTypes: {
     /**
+     * opens DropDownMenu and triggers focus of TextField
+     */
+    autoFocus: React.propTypes.bool,
```

Comment:
`openImmediately`?

I don't know why this property wasn't called `openOnMount` in SelectField, but at least we can be consistent.